resources.go

```diff
@@ -381,9 +381,16 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "autoscaling_lifecycle_hooks.html.markdown",
 				},
 			},
-			"aws_autoscaling_notification": {Tok: awsResource(autoscalingMod, "Notification")},
-			"aws_autoscaling_policy":       {Tok: awsResource(autoscalingMod, "Policy")},
-			"aws_autoscaling_schedule":     {Tok: awsResource(autoscalingMod, "Schedule")},
+			"aws_autoscaling_notification": {
+				Tok: awsResource(autoscalingMod, "Notification"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"notifications": {
+						Type: awsType(tokens.ArrayTypePrefix+autoscalingMod+"notificationType", "NotificationType"),
```

Comment:
Just for consistency with other uses of this approach - could we use this style instead:

```
awsType(apigatewayMod+"/restApi", "RestApi"),
```